lms/static/js/ccx/schedule.js

```diff
@@ -1,404 +1,385 @@
+        return node.hidden || node.category !== 'vertical';
+      });
+      this.showing = this.pruned(self.schedule, function(node) {
+        return !node.hidden;
+      });
+      this.$el.html(schedule_template({chapters: this.showing}));
+      $('table.ccx-schedule .sequential,.vertical').hide();
+      $('table.ccx-schedule .unit .toggle-collapse').on('click', this.toggle_collapse);
+      // Hidden hover fields for empty date fields
+      $('table.ccx-schedule .date button').each(function() {
+        if ($(this).text().trim() === gettext("Click to change")) {
+          $(this).html('Set date <span class="sr"> ' +
+          gettext("Click to change") + '</span>');
+        }
+      });
+
+      // Handle date edit clicks
+      $('table.ccx-schedule .date button').attr('href', '#enter-date-modal')
+      .leanModal({closeButton: '.close-modal'});
+      $('table.ccx-schedule .due-date button').on('click', this.enterNewDate('due'));
+      $('table.ccx-schedule .start-date button').on('click', this.enterNewDate('start'));
+      // click handler for expand all
+      $('#ccx_expand_all_btn').on('click', self.expandAll);
+      // click handler for collapse all
+      $('#ccx_collapse_all_btn').on('click', self.collapseAll);
+      // Click handler for remove all
+      $('table.ccx-schedule button#remove-all').on('click', function(event) {
+        event.preventDefault();
+        self.schedule_apply(self.schedule, self.hide);
+        self.dirty = true;
+        self.schedule_collection.set(self.schedule);
+        self.render();
+      });
+      // Remove unit handler
+      $('table.ccx-schedule button.remove-unit').on('click', function() {
+        var row = $(this).closest('tr'),
+        path = row.data('location').split(' '),
+        unit = self.find_unit(self.schedule, path[0], path[1], path[2]);
+        self.schedule_apply([unit], self.hide);
+        self.schedule_collection.set(self.schedule);
+        self.dirty = true;
+        self.render();
+      });
+
+      // Show or hide form
+      if (this.hidden.length) {
+        // Populate chapters select, depopulate others
+        this.chapter_select.html('')
+        .append('<option value="none">'+gettext("Select a chapter")+'...</option>')
+        .append(self.schedule_options(this.hidden));
+        this.sequential_select.html('').prop('disabled', true);
+        this.vertical_select.html('').prop('disabled', true);
+        $('form#add-unit').show();
+        $('#all-units-added').hide();
+        $('#add-unit-button').prop('disabled', true);
+      } else {
+        $('form#add-unit').hide();
+        $('#all-units-added').show();
+      }
 
+      // Show or hide save button
+      if (this.dirty) {
+        $('#dirty-schedule').show();
+        $('html, body').animate(
+          { scrollTop: $('#dirty-schedule').offset().top },
+          'slow', function() {$('#dirty-schedule').focus();
+        });
+      } else {$('#dirty-schedule').hide();}
```

Comment:
Could you put the `else` block on its own line? Even with a block of one line, it's still easier to read.
```js
} else {
  $('#dirty-schedule').hide();
}
```